### DISCUSSION TITLE
Welcome to MetricGraph Discussions!

### DISCUSSION THREAD

**Discussion by davidbolin** [davidbolin/MetricGraph#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with users of MetricGraph. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas and tips and tricks.
  * Engage with other users.
 
  To get started, comment below with an introduction of yourself and what you are using the package for.

**Comment by leninrafaelrierasegura** [comment]
Hi, I am Lenin Rafael Riera Segura, a PhD student at KAUST. I am using the package to fit a nonstationary model.

**Comment by emmaSkarstein** [comment]
Hi! I work for the Norwegian Public Roads Administration, and we are currently looking into using MetricGraph for predicting traffic volumes (annual average daily traffic) all over Norway. For our purpose, the ability to fit a directional model will be pretty important. I am still in the "figuring out what I don't know" phase, but I think we will soon have some examples running and I'm sure some questions might follow along as well. :)
